include/rangers-api/Application/Sound/CustomSound.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace app::snd {
+    class CustomSoundListener {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.